collectors/APICallCollector.js

```diff
@@ -29,17 +29,17 @@ class APICallCollector extends BaseCollector {
     async addTarget({cdpClient, url}) {
         const trackerTracker = new TrackerTracker(cdpClient.send.bind(cdpClient));
         trackerTracker.setMainURL(url.toString());
-    
+
+        cdpClient.on('Runtime.bindingCalled', this.onBindingCalled.bind(this, trackerTracker));
+        await cdpClient.send('Runtime.addBinding', {name: 'registerAPICall'});
```

Comment:
Making binding available before we create breakpoints lets us catch early calls to APIs.